integration-tests/bats/pull.bats

```diff
@@ -144,41 +144,55 @@ teardown() {
     dolt checkout feature2
     run dolt sql -q "show tables" -r csv
     [ "$status" -eq 0 ]
-    echo "$output"
     [ "${#lines[@]}" -eq 3 ]
     [[ "$output" =~ "Table" ]] || false
     [[ "$output" =~ "t1" ]] || false
     [[ "$output" =~ "t2" ]] || false
 }
 
 @test "pull: pull force" {
-    skip "todo: support dolt pull --force"
-    cd repo2
-    dolt sql -q "create table t1 (a int)"
-    dolt commit -Am "2.0 commit"
-    dolt push -f origin main
+        cd repo1
+    dolt sql <<SQL
+SET FOREIGN_KEY_CHECKS=0;
+CREATE TABLE colors (
+    id INT NOT NULL,
+    color VARCHAR(32) NOT NULL,
+
+    PRIMARY KEY (id),
+    INDEX color_index(color)
+);
+CREATE TABLE objects (
+    id INT NOT NULL,
+    name VARCHAR(64) NOT NULL,
+    color VARCHAR(32)
+);
+SQL
+    dolt commit -A -m "Commit1"
+    dolt push origin main
 
-    cd ../repo1
-    dolt sql -q "create table t2 (a int primary key)"
-    dolt sql -q "create table t3 (a int primary key)"
-    dolt commit -Am "2.1 commit"
-    dolt push -f origin main
+    cd ../repo2
+    dolt pull
+    dolt sql -q "alter table objects add constraint color FOREIGN KEY (color) REFERENCES colors(color)"
+    dolt commit -A -m "Commit2"
 
     cd ../repo1
-    run dolt pull origin
-    [ "$status" -eq 1 ]
-    [[ ! "$output" =~ "panic" ]] || false
-    [[ "$output" =~ "fetch failed; dataset head is not ancestor of commit" ]] || false
+    dolt sql -q "INSERT INTO objects (id,name,color) VALUES (1,'truck','red'),(2,'ball','green'),(3,'shoe','blue')"
+    dolt commit -A -m "Commit3"
+    dolt push origin main
 
-    dolt pull -f origin
+    cd ../repo2
+    run dolt pull
+    [ "$status" -eq 1 ]
+    [[ "$output" =~ "CONSTRAINT VIOLATION" ]] || false
 
-    run dolt log -n 1
+    dolt merge --abort
+    run dolt pull -f
```

Comment:
fully spell it out: "--force", I did a search to find it and was confused when it wasn't found